crawlers/mooncrawl/mooncrawl/cu_reports_crawler/cli.py

```diff
@@ -0,0 +1,616 @@
+        )
+
+
+def create_user_query(
+    client: Moonstream,
+    token: Union[str, UUID],
+    query_name: str,
+    query: str,
+):
+    """
+    Create a user query.
+    """
+
+    try:
+        client.create_query(token=token, name=query_name, query=query)
+    except Exception as err:
+        print(f"Cant create user query: {query_name}. End with error: {err}")
+
+
+def delete_user_query(client: Moonstream, token: str, query_name: str):
+    """
+    Delete the user's queries.
+    """
+
+    id = client.delete_query(
+        token=token,
+        name=query_name,
+    )
+
+    print(f"Query with name:{query_name} and id: {id} was deleted")
+
+
+def init_game_bank_queries_handler(args: argparse.Namespace):
+
+    """
+    Create the game bank queries.
+    """
+
+    client = Moonstream()
+
+    for query in cu_bank_queries:
+
+        try:
+            try:
+                # delete
+                delete_user_query(
+                    client=client, token=args.moonstream_token, query_name=query["name"]
+                )
+            except Exception as err:
+                print(err)
+            # create
+            created_entry = client.create_query(
+                token=args.moonstream_token,
+                name=query["name"],
+                query=query["query"],
+            )
+            print(
+                f"Created query {query['name']} please validate it in the UI url {created_entry.journal_url}/entries/{created_entry.id}/"
+            )
+        except Exception as e:
+            print(e)
+            pass
+
+
+def init_tokenomics_queries_handler(args: argparse.Namespace):
+
+    """
+    Create the tokenomics queries.
+    """
+
+    client = Moonstream()
+
+    for query in tokenomics_queries:
+
+        try:
+            try:
+                # delete
+                delete_user_query(
+                    client=client, token=args.moonstream_token, query_name=query["name"]
+                )
+            except Exception as err:
+                print(err)
+            # create
+            created_entry = client.create_query(
+                token=args.moonstream_token,
+                name=query["name"],
+                query=query["query"],
+            )
+            print(
+                f"Created query {query['name']} please validate it in the UI url {created_entry.journal_url}/entries/{created_entry.id}/"
+            )
+        except Exception as e:
+            print(e)
+            pass
+
+
+def run_tokenomics_queries_handler(args: argparse.Namespace):
+
+    client = Moonstream()
+
+    query_name = "erc20_721_volume"
+
+    ### Run voluem query
+
+    ranges = [
+        {"time_format": "YYYY-MM-DD HH24", "time_range": "24 hours"},
+        {"time_format": "YYYY-MM-DD HH24", "time_range": "7 days"},
+        {"time_format": "YYYY-MM-DD", "time_range": "30 days"},
+    ]
+
+    addresess_erc20_721 = {
```

Comment:
Should it be constant on top of file? `ADDRESSES_ERC20_721`?

Reply:
That addresses now for init tokenomics only but yes can move it on top as well as erc1155.